pkgs/applications/networking/cluster/jx/default.nix

```diff
@@ -1,49 +1,49 @@
-{ stdenv, buildGoModule, fetchFromGitHub, lib, installShellFiles }:
+{ stdenv, buildGoModule, fetchFromGitHub, lib, nix-update-script, go }:
 
 buildGoModule rec {
   pname = "jx";
-  version = "2.1.155";
+  version = "3.10.144";
```

Comment:
You might want to bump to the freshly released 3.10.146, it looks like it includes a grpc upgrade.

https://github.com/jenkins-x/jx/releases/tag/v3.10.146